css/pages/home.css

```diff
@@ -1,4 +1,21 @@
 /* base */
+@import '../base/base.css';
+@import '../base/layout.css';
+/* 3rd-party */
+@import '../3rd-party/all.min.css';
+@import url('https://fonts.googleapis.com/css2?family=Livvic:wght@300;400;500;600;700&display=swap');
```

Comment:
Everything looks fine. I see that you changed the directory in the font awesome all.min file. Just wanted to know if we really need to import google fonts if you downloaded it manually in webfonts folder?

Reply:
Good point. Actually I don't know which method is best, so we can discuss it tomorrow and I'll fix it.

Reply:
Maybe it would be effective to put @font-face to the base.css file?

Reply:
yes, `@font-face` will be at base.css, but I think that's for later. We can discuss details tomorrow, because after rechecking I saw couple of places in font awesome all.min file where directory is not changed.